docs/src/main/paradox/sqs.md

```diff
@@ -55,6 +55,23 @@ Scala
 Java
 : @@snip [snip](/sqs/src/test/java/akka/stream/alpakka/sqs/javadsl/BaseSqsTest.java) { #init-client }
 
+### Underlying HTTP client
+
+Alpakka SQS and SNS are set up to use @extref:[Akka HTTP](akka-http-docs:) as default HTTP client via the thin adapter library [AWS Akka-Http SPI implementation](https://github.com/matsluni/aws-spi-akka-http). By setting the `httpClient` explicitly (as above) the Akka actor system is reused, if not set explicitly a separate actor system will be created internally.
+
+It is possible to configure the use of Netty instead, which is Amazon's default. Please add an appropriate Netty version to the dependencies. `AmazonSQSAsyncClient` uses a fixed thread pool with 50 threads by default. To tune the thread pool used by
```

Comment:
There's no `AmazonSQSAsyncClient` anymore nor an exclusive thread pool, since the SDK 2.x uses non-blocking I/O.

Reply:
Thank you, now I changed the docs to meet reality again - I hope.